ocaml/utils/import_info.ml

```diff
@@ -14,53 +14,65 @@
 
 module CU = Compilation_unit
 
-type t =
-  | Normal of CU.t * Digest.t
-  | Normal_no_crc of CU.t
-  | Other of CU.Name.t * (CU.t * Digest.t) option
+module Intf = struct
+  type t =
+    | Normal of CU.Name.t * Digest.t (* Unpacked, so compilation unit = name *)
```

Comment:
Does this comment apply for all three cases?  If so maybe clarify

Reply:
Adding comments to the other two